cfme/tests/configure/test_visual_cloud.py

```diff
@@ -50,115 +44,118 @@ def page(request):
     return request.param
 
 
-@pytest.yield_fixture(scope="module")
-def set_grid(visual):
-    gridlimit = visual.grid_view_limit
+@pytest.yield_fixture(scope='module')
+def set_grid(appliance):
+    gridlimit = appliance.user.my_settings.visual.grid_view_limit
```

Comment:
**Optional**

Consider using `old_gridlimit` here for clarity, same for the fixtures below.